server/models/Request.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
```

Comment:
I noticed that you've branched off a previous branch to complete this feature, which is completely normal. However, when doing so, please make sure that you based your new PR on the branch you branched off of :)